modules/op-release-notes-1-2.adoc

```diff
@@ -0,0 +1,84 @@
+* The `--use-param-defaults` flag for the `tkn pipeline start` command now prompts the interactive mode if the default values have not been specified for some of the parameters.
+
+=== Triggers
+
+* The Common Expression Language (CEL) function named `parseYAML` has been added to parse a YAML string into a map of strings.
+* Error messages for parsing CEL expressions have been improved to make them more granular while evaluating expressions and when parsing the hook body for creating the evaluation environment.
+* Support for marshaling bool values and maps if they are used as the values of expressions in a CEL overlay mechanism is now available.
+* The following fields have been added to the `Eventlistener`:
+** The `replicas` field enables the event listener to run more than one pod by specifying the number of replicas in the YAML file.
+** The `NodeSelector` field enables the event listener to schedule the event listener pod to a specific node.
+* Webhook Interceptors can now parse the incoming `EventListener-Request-URL` to extract parameters from the original request URL being handled by the `EventListener`.
+* Annotations from the event listener can now be propagated to the deployment, services, and other pods. Note that custom annotations on services or deployment are overwritten, and hence, must be added to the event listener annotations so that they are propagated.
+* Proper validation for replicas in the `EventListener` specification is now available for cases when a user specifies the `spec.replicas` values as `negative` or `zero`.
+* You can now specify the `TriggerCRD` object inside the `EventListener` spec as a reference using the `TriggerRef` field to create the `TriggerCRD` separately and then bind it inside the `EventListener` spec.
+* Validation and defaults for the `TriggerCRD` object are now available.
+
+[id="deprecated-features-1-2_{context}"]
+== Deprecated features
+
+* `$(params)` are now deprecated in favor of  `$(tt.params)` to avoid confusion between the `resourcetemplate` and `triggertemplate` parameters.
+* The `ServiceAccount` reference of the optional `EventListenerTrigger` based level of authentication has changed from an object reference to a string `ServiceAccountName`. This ensures that the `ServiceAccount` is in the same namespace as the `EventListenerTrigger`.
+* The `Conditions` CRD are now deprecated, use `WhenExpressions` instead.
+* The `PipelineRun.Spec.ServiceAccountNames` is being deprecated in favor of `PipelineRun.Spec.TaskRunSpec[].ServiceAccountName`.
+
+[id="known-issues-1-2_{context}"]
+== Known issues
+
+* This release of {pipelines-title} adds support for a disconnected installation. However, some cluster tasks need to get mirrored in order to work in disconnected clusters.
+* Uninstalling the {pipelines-title} Operator does not remove the event listeners. To work around this, add the `finalizer` manually or make an API call with the foreground delete option set.
```

Comment:
@savitaashture @khrm is this enough or do we need to provide the command as well?

Reply:
How about specifying the k8s link where the examples are given
https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#setting-the-cascading-deletion-policy

WDYT @khrm ?

Reply:
We should just use command here like in the point next to this.

Reply:
Workaround to remove EventListener CRD and Pod is
1. User has to edit eventlistener object with below finalizers
```
finalizers:
  - foregrounddeletion
```
2. execute
`kubectl patch crd/eventlisteners.triggers.tekton.dev -p '{"metadata":{"finalizers":[]}}' --type=merge` to delete EventListener CRD

Reply:
Fixed